src/style.css

```
@@ -1,3 +1,107 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+li {
+  margin-left: 20px;
+}
+
+/* Use Roboto font */
+body {
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+}
+
+/* Header styling */
+header {
+  background-color: #fff;
+  justify-content: space-between;
+  width: 100%;
+  display: flex;
+  align-items: center;
+  padding: 0 50px;
+}
+
+/* Logo styling */
+.logo img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+/* Navigation styling */
+.nav-list {
+  display: flex;
+  list-style: none;
+}
+
+.nav-item {
+  display: flex;
+  justify-content: center; /* Центрування по горизонталі */
+  align-items: center; /* Центрування по вертикалі */
+  margin-left: 20px;
+}
+
+.nav-item:first-child {
+  margin-left: 0;
+}
+
+.nav-item:last-child {
+  display: flex;
+  justify-content: center; /* Центрування по горизонталі */
+  align-items: center; /* Центрування по вертикалі */
+}
+
+.nav-link {
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  line-height: 60px; /* Додано: Ensuring the link height matches the header height */
+  color: black; /* Default link color */
+  text-decoration: none;
+  position: relative;
+  display: inline-block; /* Додано: Ensure the links are treated as blocks */
+  text-align: left;
+}
+
+.is-active::after {
+  content: '';
+  display: block;
+  width: 100%;
+  height: 4px;
+  background-color: #00acdc; /* Голуба лінія під активним посиланням */
+  position: absolute;
+  left: 0;
+  border-radius: 8px; /* Закруглені краї */
+  margin-bottom: 0;
+  margin-top: -4px;
+}
+
+/* Style for blue text on hover for active link */
+.nav-link.is-active {
+  color: #00acdc; /* Change text color on hover */
+}
+
+.nav-link:hover {
+  color: #00acdc;
+}
```

```suggestion
.nav-link.is-active,
.nav-link:hover {
  color: #00acdc;
}
```
you can join selectors with same styles